CHANGELOG.md

```diff
@@ -24,6 +24,21 @@ The following API changes should be documented as such:
 If possible, provide tooling that performs the changes, e.g. a shell-script.
 -->
 
+# 3.2.0
+
+## Notable Bug-fixes
+
+#### Utility
+
+* `seqan3::views::single_pass_input` can't keep the `std::ranges::output_range` property, because it can't satisfy the following property:
```

Comment:
```suggestion
* `seqan3::views::single_pass_input` cannot propagate the `std::ranges::output_range` property, because it cannot 
  satisfy the following requirement ([\#2775](https://github.com/seqan/seqan3/pull/2775)):
```